css/style.css

```diff
@@ -126,21 +126,26 @@ button:hover {
 	position: relative;
 	z-index: 120;
 	height: 44px;
+	display: flex;
+	align-items: center;
 }
 
 #board-header > * {
 	font-size: 14pt;
```

Comment:
@artemanufrij I meant that the font-size declaration can also be moved over to there